canonical-web-and-design/vanilla-framework
# Move card padding to `p-card__inner`

## Done

- Removed padding from `p-card` elements, and added it to a new `p-card__inner` element
- Created "Content Bleed" card example
- Updated card markup across the site, including examples
  - Refactored icon docs to make it easier to do the above for 100s of icons

Fixes #971

## QA

- Open [demo](https://vanilla-framework-3953.demos.haus/docs/examples/patterns/card/default)
- See that the pattern looks the same as on live (https://vanillaframework.io/docs/examples/patterns/card/default)
- Open [content bleed example](https://vanilla-framework-3953.demos.haus/docs/examples/patterns/card/content-bleed)
- See that it looks good
- Review updated documentation:
  - https://vanilla-framework-3953.demos.haus/docs/upgrade-guide-v3
  - https://vanilla-framework-3953.demos.haus/docs/patterns/card

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical-web-and-design/vanilla-framework/wiki/Release-process#pre-release-tasks)
